.github/workflows/ubuntu-minimal.yml

```diff
@@ -0,0 +1,34 @@
+name: Mathics (ubuntu-minimal)
```

Comment:
This workflow was added to check that indeed with the changes the tests pass with a minimal installation.